src/Integration/Integration.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="BouncyCastle" Version="1.8.6.1" />
```

Comment:
I think it's worth including a comment about why we use this package, and also that we only use the MD5 algorithm.
The package is 2.5M and it increases the size of the VSIX from 32.9M to 35.4M, so it's a not insignificant increase. If we need to reduce the size of the VSIX in future then finding an alternative package (or just extracting the bits we need, if that's ok from a licensing point of view) would be an option.